libopenage/util/compiler.cpp

```diff
@@ -63,8 +122,8 @@ std::string symbol_name(const void *addr, bool require_exact_addr, bool no_pure_
 			return demangle(addr_info.dli_sname);
 		} else {
 			return util::sformat("%s+0x%lx",
-			                     demangle(addr_info.dli_sname).c_str(),
-			                     symbol_offset);
+				demangle(addr_info.dli_sname).c_str(),
+				symbol_offset);
```

Comment:
the indent here was alright :)  tabs to reach the indent level of `return`, then spaces to align to the `(`.